api/RTCRtpContributingSource.json

```diff
@@ -80,26 +80,38 @@
             "edge_mobile": {
               "version_added": null
             },
-            "firefox": {
-              "version_added": "59",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "media.peerconnection.rtpsourcesapi.enable",
-                  "value_to_set": "true"
-                }
-              ]
-            },
-            "firefox_android": {
-              "version_added": "59",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "media.peerconnection.rtpsourcesapi.enable",
-                  "value_to_set": "true"
-                }
-              ]
-            },
+            "firefox": [
+              {
+                "version_added": "59",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "media.peerconnection.rtpsourcesapi.enable",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
```

Comment:
This block, as described, means that the preference has been removed. According to the bug, this is not the case. Please remove it.

Reply:
According to [1]. this seems to be the correct way to insert information about a particular browser version's support level.

[1] https://developer.mozilla.org/en-US/docs/MDN/Contribute/Structures/Compatibility_tables#Including_multiple_support_points_for_the_same_browser_entry